src/Components/AffectedClustersTable/AffectedClustersTable.js

```diff
@@ -0,0 +1,257 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import PrimaryToolbar from '@redhat-cloud-services/frontend-components/PrimaryToolbar';
+import { conditionalFilterType } from '@redhat-cloud-services/frontend-components/ConditionalFilter/conditionalFilterConstants';
+import { EmptyTable } from '@redhat-cloud-services/frontend-components/EmptyTable';
+import { TableToolbar } from '@redhat-cloud-services/frontend-components/TableToolbar';
+
+import { global_success_color_100 as globalSuccessColor100 } from '@patternfly/react-tokens/dist/js/global_success_color_100';
+import { global_danger_color_100 as globalDangerColor100 } from '@patternfly/react-tokens/dist/js/global_danger_color_100';
+import CheckCircleIcon from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import { Card, CardBody } from '@patternfly/react-core/dist/js/components/Card';
+import { sortable } from '@patternfly/react-table/dist/js/components/Table/utils/decorators/sortable';
+import { Table } from '@patternfly/react-table/dist/js/components/Table/Table';
+import { TableBody } from '@patternfly/react-table/dist/js/components/Table/Body';
+import { TableHeader } from '@patternfly/react-table/dist/js/components/Table/Header';
+import { Bullseye } from '@patternfly/react-core/dist/js/layouts/Bullseye';
+import {
+  EmptyState,
+  EmptyStateVariant,
+} from '@patternfly/react-core/dist/js/components/EmptyState/EmptyState';
+import { EmptyStateBody } from '@patternfly/react-core/dist/js/components/EmptyState';
+import {
+  Pagination,
+  PaginationVariant,
+} from '@patternfly/react-core/dist/js/components/Pagination/Pagination';
+import { Title } from '@patternfly/react-core/dist/js/components/Title';
+
+import messages from '../../Messages';
+import MessageState from '../MessageState/MessageState';
+import Loading from '../Loading/Loading';
+
+const AffectedClustersTable = ({ affectedClusters }) => {
+  const intl = useIntl();
+  const { isError, isUninitialized, isLoading, isFetching, isSuccess, data } =
+    affectedClusters;
+  const rows = data?.data || [];
+  const [activeFilters, setActiveFilters] = useState({ name: '' });
```

Comment:
notta requirement, but this is ah BUNCH of usestates, might want to look into using useReducer instead... (

Reply:
@AllenBW , really good point, refactored it at https://github.com/RedHatInsights/ocp-advisor-frontend/pull/32/commits/7ef565eea601bd4e825a5eb786f32b1069f91527